### ISSUE TITLE
You can scoop ash with welder

### ISSUE THREAD

**Issue by SoulFN** [space-wizards/space-station-14#30234]
## Description

See description
**Reproduction**

Click on ash pillow with welding tool
**Screenshots**

No
**Additional context**

No

**Comment by Cojoke-dot** [comment]
yeah... it seems you can just put anything into a welder... I thought there was a whitelist or something on reagents for them but I guess not?

**Comment by baddiePRO** [comment]
> yeah... it seems you can just put anything into a welder... 

does this mean you can weld with any reagent or will just welding fuel work?

**Comment by Errant-4** [comment]
> > yeah... it seems you can just put anything into a welder...
> 
> does this mean you can weld with any reagent or will just welding fuel work?

It does not refuel the welder

**Comment by Nukesthestation** [comment]
if this is still an issue, i can try and tackle it
